### DISCUSSION TITLE
Some comments about this great Mod (fun-bots) for BF3 (VU)

### DISCUSSION THREAD

**Discussion by gtpeter** [Joe91/fun-bots#203] [General]
Hi, Joe

I am very basic BF player, hence I am quite frustrated with online playing with experienced players. I feel that I am just the tools for them to earn more XP etc. (yeah, a bit more pessimistic to groan in that way :P)

With born of VU and your fun-bots, I really enjoy the exploration inside different maps and engaging enemies with more Military Realistic rhythm (Yet still too fast and heroic in comparison with e.g. ArmA 3)

Anyway, after playing with VU and fun-bots for about 3 weeks, these are what I found:

1.   There is 30 seconds pre-round waiting.  I couldn't make any change on that.  I don't play with other real players, hence this 30s waiting might be too much for me alone.   Of course,  I did guess it should be controlled by VU setting, but vars.roundLockdownCountdown doesn't seem work.   Thus, I am wondering if there is any fun-bots setting overrides it ?

2.   Frankly and generally speaking, under default fun-bots settings (except I increase bots to 16 per team), I noted either bots shot me or I shot bots, we have to get about 10 hits on the body before die (except head-shot).   Personally I felt a little too long, but still ok, because I am still learning, and I could have ample response-time to identify where is the shot coming from.     Yesterday, I joined a COOP BF VU server with fun-bots and other 6-7 real players,  both bots or I could die with 2-3 shots.  I understand this should be due to server setting.  The problem is, after I return to my own local server, this doesn't change back to 10-shot.  With 2-3 shots to die, I don't even have response time to take cover or know where is the bot. The experience is almost like in the official BF3 server confronting with experienced players.  The most-strange part is, when I check the fun-bots/ext/shared/config.lua file,  there is no change of setting (except 16-bots per team), and the file timestamp remains about 1-2 week ago. 

3.  The bots chase/flow each other in circle (even within the same team) has improved from v. 2.2.2 to 2.30,  but still existing.   If I saw my own team Bots go circle,  normally I would walk through to break the circle, after that, the bots would head to the target location.

Thanks for the great mod, wish to see more improvement with new update :) +1:

**Comment by OverlordFreddo** [comment]
> 1. There is 30 seconds pre-round waiting.  I couldn't make any change on that.

There is a mod called NoPreRound that should take care of that - <https://github.com/FlashHit/VU-Mods/tree/master/NoPreRound>

Just download the lot as a zip and move the NoPreRound folder into the mods folder and there will be no waiting time.

**Comment by Joe91** [comment]
In the settings you can modify the difficulzy. And in the new dev-Version your issue should be gone.

**Comment by gtpeter** [comment]
Thanks so much,  OverlordFreddo!  This "NoPreRound" Mod work perfect :)

And Joe,  I changed some difficulties in config.lua file, e.g. from 0.2 --> 0.5,  0.5 -->0.8.    Yes, it seems slightly better (about 20%), but not back to previous status.  I would continue to tweak around.   Thanks very much :+1:
